src/util/formatter.ts

```diff
@@ -34,6 +35,30 @@ export class OutputFormatter {
 		this.write(obj, options, true);
 	}
 
+	async writeAllPages<T>(func: (params: any) => Promise<PagedResult<T>>, params?: any, options?: OutputOptions): Promise<void> {
```

Comment:
`writeAllPages` doesn't seem to be used anywhere (I couldn't find any references to it).  If so, please remove.